cmd/backup-operator/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The etcd-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// backup-operator is an example implementation of backing up an etcd cluster to S3.
+// user must not depend backup-operator for production usage and should
+// follow the design principle of backup-operator to create his/her own backup
+// machanism.
```

Comment:
mechanism

Same for restore-operator doc comment.

Reply:
good catch